templates/Dockerfile-run

```diff
@@ -309,6 +309,33 @@ RUN \
         make install && \
         rm -rf ${DIR}
 
+## libxml2 - for libbluray
+RUN \
+        DIR=/tmp/libxml2 && \
+        mkdir -p ${DIR} && \
+        cd ${DIR} && \
+        curl -sLO http://xmlsoft.org/sources/libxml2-${LIBXML2_VERSION}.tar.gz && \
```

Comment:
https://gitlab.gnome.org/GNOME/libxml2/-/tags/ might be a better source for the code

Reply:
👍 never occurred to me to go find a more recent version, I've just had xmlsoft.org written down as "the place to get libxml2" for so long that I didn't even think about the actual source repository 🙂

Have updated this.